swagger.yaml

```diff
@@ -3048,12 +3048,39 @@ paths:
               name:
                 type: string
                 description: Name of the Category
+              description:
+                type: string
+                description: Description of the Category
               path:
                 type: string
               uuid:
                 type: string
                 description: The unique identifier
                 format: uuid
+              statistics:
+                type: object
+                properties:
+                  topic_count:
+                    type: integer
+                  follower_count:
+                    type: integer
+                  topic_type_count:
+                    type: object
+                    properties:
+                      question:
+                        type: integer
+                      feedback:
+                        type: integer
+                      article:
+                        type: integer
+                      tutorial:
+                        type: integer
+                      faq:
+                        type: integer
+              topic:
+                $ref: '#/components/schemas/TopicsResponse'
```

Comment:
The response here doesn't return the topics response, it doesn't have the total count 
the response would be an array of topicResponse or Topic